odl/contrib/fom/supervised/supervised_fom.py

```diff
@@ -0,0 +1,570 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Implementation of Figure of Merits (FOMs) for comparing reconstructions with
+a given reference.
+"""
+
+__all__ = ('mean_square_error', 'mean_absolute_error', 'mean_value_difference',
+           'standard_deviation_difference', 'range_difference', 'blurring',
+           'false_structures')
+
+
+def mean_square_error(test_data, ref_data, mask=None, normalized=False):
+    """Mean square error-based FOM between ``test_data`` and ``ref_data``.
+
+    Evaluates mean square error between input (``test_data``) and
+    reference data (``ref_data``), by measuring consistency using the L2 -norm.
+    Allows for normalization (``normalized``) and a masking of the two spaces
+    (``mask``).
```

Comment:
This doesn't really add any information. If there's nothing more to say that's not apparent from signature, one-line summary and `Parameters`, there's no need to write anything.